server/tests/template.load

```diff
@@ -14,7 +16,10 @@ SET MySQL PARAMETERS
       net_read_timeout  = '120',
       net_write_timeout = '120'
 
-CAST column Drafts.Priority to text,
+CAST column Channels.Type to "channel_type" drop typemod,
+     column Teams.Type to "team_type" drop typemod,
+     column UploadSessions.Type to "upload_session_type" drop typemod,
+     column Drafts.Priority to text,
```

Comment:
Would this mean that new tables added in the schema in future would need entries here as well? Would tests fail if we fail to do that?

Reply:
This is just for convenience for the enums, pgloader becoming very verbose if these typecasting rules are not added. Otherweise the result would be the same, so the tests won't fail as long as the underlying data doesn't change after the migration.

Reply:
Ah okay